### PR TITLE
Individual video page

### PR DESCRIPTION
**Pull request by abhishektanwar**
- Fixed user authentication after reloading
- Added video play page for each video

Files to check:
- src/Contexts/AuthDialogContext/index.jsx
- src/components/VideoCard/index.jsx
- src/components/VideoPlayer/index.jsx
- src/components/VideoPlayer/video-player.css
- src/screens/VideoPage/index.jsx

### CODE REVIEW

File: src/Contexts/AuthDialogContext/index.jsx [hunk 1]
```diff
@@ -24,50 +27,81 @@ export const AuthProvider = (props) => {
...
         password: user.password,
-      })
-      if(res.statusText==='Created'){
-        setUser(res.data.createdUser)
-        utils.setLocalStorage('authToken',res.data.encodedToken);
-        setError({errorExists:false,errorMessage:''})
-        setIsLoading(false)
+      });
+      if (res.statusText === "Created") {
+        // setUser(res.data.createdUser)
```
Comment (sanjitsarkar) [inline src/Contexts/AuthDialogContext/index.jsx]: Don't use cosole.log in production

**Review by Ranjeetk489 — APPROVED** [review]
suggestion: it'll look good if video card has some padding from right 
- code is well structured, good to merge

File: src/Contexts/AuthDialogContext/index.jsx [hunk 2]
```diff
@@ -24,50 +27,81 @@ export const AuthProvider = (props) => {
...
+          encodedToken: res.data.encodedToken,
+        });
+        utils.setLocalStorage("authToken", res.data.encodedToken);
+        setError({ errorExists: false, errorMessage: "" });
+        setIsLoading(false);
         return true;
       }
-    }
-    catch(e){
-      setError({errorExists:true,errorMessage:"Something went wront. Please try again"});
```
Comment (sanjitsarkar) [inline src/Contexts/AuthDialogContext/index.jsx:62]: Give a proper error message instead of this

File: src/App.js [hunk 3]
```diff
@@ -31,7 +32,9 @@ function App() {
                   <Route path="/liked-videos" element={<LikedVideos />} />
                   <Route path="/history" element={<History />} />
                   <Route path="/playlists" element={<Playlists />} />
-                  <Route path="/video" element={<VideoPage />} />
+                  {/* <Route path="/video" element={<VideoPage />} /> */}
```
Comment (iRohitGaur) [inline src/App.js]: suggestion: remove commented code

File: src/Contexts/AuthDialogContext/index.jsx [hunk 4]
```diff
@@ -24,50 +27,81 @@ export const AuthProvider = (props) => {
...
+    } catch (e) {
+      setError({
+        errorExists: true,
```
Comment (sanjitsarkar) [inline src/Contexts/AuthDialogContext/index.jsx:75]: if there is no error simply keep the errorMessage to empty string.
There is no need of using errorExists

**Review by sanjitsarkar — APPROVED** [review]

File: src/Contexts/AuthDialogContext/index.jsx (hunk 1 above)
Comment (iRohitGaur) [inline src/Contexts/AuthDialogContext/index.jsx]: suggestion: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/App.js (hunk 3 above)
Comment (abhishektanwar) [reply src/App.js]: removed

File: src/Contexts/AuthDialogContext/index.jsx (hunk 1 above)
Comment (abhishektanwar) [reply src/Contexts/AuthDialogContext/index.jsx]: removed

File: src/Contexts/AuthDialogContext/index.jsx (hunk 4 above)
Comment (abhishektanwar) [reply src/Contexts/AuthDialogContext/index.jsx:75]: thanks for the suggestion , I will keep this in mind moving forward

File: src/Contexts/AuthDialogContext/index.jsx (hunk 1 above)
Comment (abhishektanwar) [reply src/Contexts/AuthDialogContext/index.jsx]: removed

**Pull request merged**